assets/src/js/query_loop_carousel.js

```diff
@@ -1,8 +1,16 @@
 import {v4 as uuid} from 'uuid';
 
+const ARROW_DIRECTIONS = ['prev', 'next'];
+
+const removeArrows = layout => ARROW_DIRECTIONS.forEach(direction => {
+  const controlBtn = layout.querySelector(`.wp-block-buttons .carousel-control-${direction}`);
```

Comment:
Should we add an early return if `layout` is falsy?

Reply:
It should be okay, I've added a check for `layout` in the main function and we only use this one there